test/types/objectSearch.test.tsx

```diff
@@ -0,0 +1,68 @@
+import faker from 'faker';
+
+import { Tester } from '@mighty-justice/tester';
+
+import { fillInFieldConfig } from '../../src';
+import ObjectSearch from '../../src/inputs/ObjectSearch';
+
+function getDefaults (overrides?: any) {
+  const fakeLawFirm = () => ({
+      id: faker.random.uuid(),
+      name: faker.company.companyName(),
+    })
+    , field = overrides.field || 'law_firm'
+    , endpoint = overrides.endpoint || 'legal-organizations'
+    , type = overrides.type || 'objectSearch'
+    , fieldConfig = fillInFieldConfig({ field, type, endpoint, ...overrides.fieldConfig })
+    , props = { id: field, fieldConfig, debounceWait: 0 }
+    ;
+
+  return {
+    result: fakeLawFirm(),
+    searchTerm: faker.lorem.sentence(),
+
+    endpoint,
+    field,
+    fieldConfig,
+    props,
+    type,
+
+    ...overrides,
+  };
+}
+
+async function searchFor (tester: any, field: string, result: any, searchTerm: string) {
```

Comment:
This function is copied and pasted from [here](https://github.com/mighty-justice/fields-ant/blob/a8802f0f2325fd15105136fdfde6ea48b5f90498/test/types/objectSearchCreate.test.tsx#L48). I know these are specific to testing OSC components, but may as well import from this file or vice versa